sklearn/census_online_prediction.py

```diff
@@ -0,0 +1,177 @@
+# [START imports]
+import googleapiclient.discovery
+import json
+import numpy as np
+import os
+import pandas as pd
+import pickle
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.externals import joblib
+from sklearn.feature_selection import SelectKBest
+from sklearn.pipeline import FeatureUnion
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import LabelBinarizer
+# [END imports]
+
+# [START setup]
```

Comment:
Move the #[START setup] down one line so there is no space, but still have 2 spaces between. SO:

# [END imports]


# [START setup]
# Define variables